content/02.main-text.md

```diff
@@ -228,11 +233,12 @@ Examples of continuous integration to automate manuscript generation include [gh
 Distill journal articles [@doi:10.23915/distill.00010], Idyll [@tag:idyll], and Stencila [@tag:stencila] support manuscripts with interactive graphics and close integration with the underlying code.
 As an open source project, Manubot can be extended to adopt best practices from these other emerging platforms.
 
-Many others have embraced open science principles and piloted open approaches toward drug discovery [@doi:10.7554/eLife.26726; @doi:10.1371/journal.pmed.1002276], data management [@doi:10.1371/journal.pbio.1001195; @doi:10.1038/sdata.2016.18; @doi:10.1038/nbt.3516], and manuscript review [@doi:10.12688/f1000research.12037.1].
-Several of these open science efforts are GitHub-based like our collaborative writing process.
+Several open science efforts are GitHub-based like our collaborative writing process.
```

Comment:
You moved `doi:10.12688/f1000research.12037.1` to a section below, and it fits well there.  If for any reason we remove that reference below, it would also fit here because of the section on GitHub-based review.